audit-service/README.md

```diff
@@ -18,8 +18,15 @@ limitations under the License.
 
 # Audit Service
 
-The Audit service accepts incoming messages on the `error` and `success` Kafka topics, these messages contain all the details of the initial request and any other relevant information. This information will be passed to any local audit services that have
-been implemented. This service does not have any output Kafka topics.
+The Audit service accepts incoming messages on the `error` and `success` Kafka topics, these messages contain all the details of the initial request and any other relevant information. 
```

Comment:
I think it needs a rewording.  The "messages contain all the details of the initial request and any other relevant information" is vague and arguably wrong.  Suggest something in your own word along the lines of "The message contains information about the  request and in the case of an error, information about the error".

Reply:
This has been re-worded